.github/workflows/generate-page.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and deploy translation status to Github Pages
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - Language/**
+      - tests/Language/**
```

Comment:
I'm not sure if we ever accepted language translation without tests. But if we did (and allowed tests to be added later), that would be a valid reason to have this line.

We rely on tests to build the list of available languages.

Reply:
The only test file we should be watching here is `AbstractTranslationTestCase` because we use the `$locales` in the script. For other tests I don't think there's much to watch.
```suggestion
      - tests/Language/AbstractTranslationTestCase.php
```

Reply:
Fixed.